pkg/virt-handler/migration-source.go

```diff
@@ -615,5 +615,9 @@ func configureParallelMigrationThreads(options *cmdclient.MigrationOptions, vm *
 		return
 	}
 
+	if options.AllowPostCopy {
+		return
+	}
```

Comment:
Thanks @fossedihelm!

I must be missing something, but I'm just not sure how this solves the problem this PR outlines: `"With same version of handler and launcher, everything is working well, but during upgrades, the launcher version is different from the handler one"`.

Let's say that post-copy is not allowed, isn't the problem you've mention also exists? how does it solve it in this case?

Reply:
I am not sure I understand what you mean, but if postcopy is not enabled, multifd can be used without any issue.

Reply:
@iholder101 Sure

Reply:
As per https://github.com/kubevirt/kubevirt/pull/15015#discussion_r2168914019
The idea of the revert was reverted for the moment :D